This got pinned upstream to avoid pulling in a bundled openssl 3.x that causes
problem on some android targets..

see https://github.com/sfackler/rust-openssl/issues/2163

Index: cargo/Cargo.toml
===================================================================
--- cargo.orig/Cargo.toml
+++ cargo/Cargo.toml
@@ -326,7 +326,7 @@ version = "0.4.42"
 default-features = false
 
 [target."cfg(not(windows))".dependencies.openssl]
-version = "=0.10.57"
+version = "0.10.57"
 optional = true
 
 [target.'cfg(target_os = "linux")'.dependencies.cargo-credential-libsecret]
